main.py

```
@@ -1,279 +1,144 @@
-async def ping_slash(interaction: discord.Interaction):
-    """Responds with Pong! and the bot's latency."""
-    latency_ms = round(bot.latency * 1000)
-    await interaction.response.send_message(f"Pong! Latency: {latency_ms}ms")
+    """
+    Event handler called when the bot has successfully connected to Discord,
+    finished preparations, and is ready to operate.
+    This event may be called multiple times if the bot disconnects and reconnects.
+    """
+    print("-" * 50)
+    print(f"🎉 Bot logged in as {bot.user.name} (ID: {bot.user.id})")
+    print(f"🔑 Successfully connected to Discord services.")
```

Ruff (0.11.9)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)